test/device-test/src/framework/test_common.c

```diff
@@ -318,7 +318,9 @@ static void read_and_print2screen(struct test_usb_metadata* metadata)
 
     int ret = read_single_frame((USB_FRAME*)metadata->usb_hid_out_report, &metadata->packet);
     if (ret == 0) {
-        screen_print_debug(data_to_string(&metadata->packet), 1000);
+        char* tmp = data_to_string(&metadata->packet);
```

Comment:
`data_to_string` could also return a static string, so using it keeps being a one-liner.

Reply:
ack